### PR TITLE
DWR-1627 Aggregate service - more test coverage.

### PR DESCRIPTION
**Pull request by agorina**
Still not 100% coverage, but a step closer to it.

### CODE REVIEW

File: aggregate-service/src/test/java/org/opentestsystem/rdw/olap/service/impl/DefaultCustomAggregateServiceRST.java [hunk 1]
```diff
@@ -1,910 +0,0 @@
...
-import static org.opentestsystem.rdw.reporting.common.model.DimensionType.Section504;
-import static org.opentestsystem.rdw.reporting.common.model.DimensionType.StudentEnrolledGrade;
-import static org.opentestsystem.rdw.reporting.common.model.OrganizationType.State;
-import static org.opentestsystem.rdw.reporting.common.test.support.PermissionScopes.permissions;
-import static org.springframework.test.context.jdbc.Sql.ExecutionPhase.AFTER_TEST_METHOD;
-import static org.springframework.test.context.jdbc.Sql.ExecutionPhase.BEFORE_TEST_METHOD;
-import static org.springframework.test.jdbc.JdbcTestUtils.countRowsInTable;
-
-//TODO: clean it up before completing aggregate repository refactoring
-@Ignore
```
Comment (agorina) [inline aggregate-service/src/test/java/org/opentestsystem/rdw/olap/service/impl/DefaultCustomAggregateServiceRST.java]: As promised, this is gone.

**Comment by agorina** [comment]
To avoid a merge conflict with @wtritch's work tomorrow, I am merging it now but will address comments (if needed).

File: aggregate-service/src/main/java/org/opentestsystem/rdw/olap/model/OrganizationTypeQuery.java [hunk 2]
```diff
@@ -104,6 +105,37 @@ public static Builder builder() {
...
+                && Objects.equals(organizationType, that.organizationType)
+                && Objects.equals(completenessCodes, that.completenessCodes)
+                && Objects.equals(administrativeConditionCodes, that.administrativeConditionCodes)
+                && Objects.equals(genderCodes, that.genderCodes)
+                && Objects.equals(ethnicityCodes, that.ethnicityCodes)
+                && Objects.equals(lepCodes, that.lepCodes)
+                && Objects.equals(migrantStatusCodes, that.migrantStatusCodes)
+                && Objects.equals(section504Codes, that.section504Codes)
+                && Objects.equals(iepCodes, that.iepCodes)
+                && Objects.equals(economicDisadvantageCodes, that.economicDisadvantageCodes);
```
Comment (agorina) [inline aggregate-service/src/main/java/org/opentestsystem/rdw/olap/model/OrganizationTypeQuery.java]: Needed for tests:
` when(repository.findByOrganizationType(eq(user), eq(districtQuery), eq(OrganizationType.District))).thenReturn(districtReport);`

Comment (malaffoon) [reply aggregate-service/src/main/java/org/opentestsystem/rdw/olap/model/OrganizationTypeQuery.java]: there is that unit test compare by field thing but i'm fine with this

Comment (agorina) [reply aggregate-service/src/main/java/org/opentestsystem/rdw/olap/model/OrganizationTypeQuery.java]: I am using it in the cases `assertThat(..).usingRecursiveFieldByFieldElementComparator()` or `isEqualToComparingFieldByFieldRecursively`, but was not sure how to us it in the above example. Do you have a sample code?

**Pull request merged**